network-api/networkapi/wagtailpages/templatetags/multi_image_tags.py

```diff
@@ -8,3 +8,16 @@ def optimize_images(image):
     return {
         'unoptimized_image': image
     }
+
+
+@register.simple_tag(name='custom_image_height')
+def custom_image_height(image, height=410):
+    """
+    Automatically fill an image with 1400 by X pixels where X is a variable integer.
+
+    If a height is not set (False or None) default to a 410px height as per the
+    original design.
+    """
+    if not height:
+        height = 410
```

Comment:
this is redundant: if `height` is not passed, it will already be `410` because of the argument signature.
```suggestion
```

Reply:
Yup, because I iterated on it and didnt change it haha